Shared/mods/deathmatch/logic/Utils.h

```diff
@@ -336,3 +336,9 @@ void DeletePointersAndClearList(T& elementList)
         delete *iter;
     }
 }
+
+// Comparing floating point numbers
+bool approximatelyEqual(float a, float b, float epsilon = FLOAT_EPSILON);
```

Comment:
Why do these names start with lower case letters?

Reply:
I copied it from the mentioned PR and didn’t pay attention